test/accel/IntegrationTestBase.cc

```diff
@@ -497,6 +497,50 @@ auto TestEm3IntegrationMixin::make_sens_det(std::string const& sd_name)
     return std::make_unique<SimpleSensitiveDetector>(sd_name);
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Create physics list: default is EM only using make_physics_input.
+ */
+auto OpNoviceIntegrationMixin::make_physics_input() const -> PhysicsInput
+{
+    PhysicsInput result = Base::make_physics_input();
+    result.em_bins_per_decade = 5;
+    return result;
```

Comment:
I think we want OpNovice to *always* have optical physics, so you'll need to copy/adapt the code from `LarSphereOptical::make_physics_input` (from `TrackingManagerIntegration`) here and in `make_setup_options`.